src/System.Net.WebSockets.Client/tests/CancelTest.cs

```diff
@@ -125,19 +124,12 @@ public class CancelTest : ClientWebSocketTestBase
         {
             using (ClientWebSocket cws = await WebSocketHelper.GetConnectedWebSocket(server, TimeOutMilliseconds, _output))
             {
-                var cts = new CancellationTokenSource(500);
-
                 var recvBuffer = new byte[100];
                 var segment = new ArraySegment<byte>(recvBuffer);
 
-                try
-                {
-                    await cws.ReceiveAsync(segment, cts.Token);
-                    Assert.True(false, "Receive should not complete.");
-                }
-                catch (OperationCanceledException) { }
-                catch (ObjectDisposedException) { }
-                catch (WebSocketException) { }
+                var cts = new CancellationTokenSource(100);
+                // If this fails with OperationCancelledException, it was not yet being awaited - rerun test.
```

Comment:
> 100 ms is an eternity to a machine

This is incorrect for most of our CI test machines as well as most Azure's A1s and A4s. Unless you're willing to test on those, please keep the current timeout.